fmriprep/cli/run.py

```diff
@@ -145,6 +145,9 @@ def get_parser():
     g_aroma = parser.add_argument_group('Specific options for running ICA_AROMA')
     g_aroma.add_argument('--use-aroma', action='store_true', default=False,
                          help='add ICA_AROMA to your preprocessing stream')
+    g_aroma.add_argument('--return-non-denoised', action='store_true', default=False,
```

Comment:
added this option in case a user want to return both the denoised and the non-denoised confounds/files